src/main/java/tools/jackson/databind/introspect/POJOPropertiesCollector.java

```diff
@@ -702,7 +702,12 @@ protected void _addCreators(Map<String, POJOPropertyBuilder> props)
             // But only if no Default (0-args) constructor available OR if we are configured
```

Comment:
@JooHyukKim this would explicitly prevent detection, as designed.

To me the bigger question is why doesn't 2.18 or 2.19 prevent detection too?